frontend/src/components/compiler/CompilerOpts.tsx

```diff
@@ -52,7 +56,7 @@ export function FlagOption({ flag, description }: { flag: string, description?:
         value={flag}
         selected={checkFlag(flag)}
     >
-        {flag} {description && `(${description})`}
+        {flag} {description && description !== "NULL" && `(${description})`}
```

Comment:
we can't return the empty string by default or `null`, so this is what I settled on. thoughts?

Reply:
is fine with me